modules/react/select/lib/hooks/useSelectInput.ts

```diff
@@ -130,6 +132,11 @@ export const useSelectInput = composeHooks(
         },
         textInputProps: {
           ref: textInputRef,
+          value: elemProps.value
+            ? model.navigation.getItem(model.state.selectedIds[0], model).textValue
+            : model.state.selectedIds.length > 0 && model.state.items.length > 0
```

Comment:
```suggestion
            : model.state.selectedIds.length && model.state.items.length
```

Reply:
What is the failure of the ternary supposed to do? The first part says "if the developer passes a `value`, do a text lookup of that value". The original sandbox had the failure case return `undefined`, meaning "if the developer didn't pass a `value`, don't treat this like a controllable input". But the `model.state.selectedIds.length > 0 && model.state.items.length > 0` will return `true` or `false` if a `value` is not provided. What does this do? It doesn't match the original intent, but what is the side effect?